components/budget/budget_form.js

```diff
@@ -0,0 +1,69 @@
+import {createElement} from "../../commons.js";
+import { Budget } from "./budget.js";
+
+export class BudgetForm {
+
+    constructor(selector, budget) {
+
+        this.element = document.querySelector('.budget-form-container');
+        this.budget = budget;
+        this.budgetForm  = createElement("form", {class: selector, onsubmit:"return false;"});
+        this.container = createElement("div", {class: "container"});
+        this.BudgetAndExpensesComponent = createElement("div", {class: "Budget_And_Expenses"});
+
+        this.createForm();
+        this.createButtons();
+        this.sendBudgetValueToBudgetComponent();
+    }
+
+    createForm() {
+         this.budget.find(item => {
+            if(item.type === 'number') {
+                this.createLablesAndInputs(item);
+
+            } else if(item.type === 'text') {
+                this.createLablesAndInputs(item);
+            }
+          })
+          document.querySelector(".budget-app").append(this.container);
+          
+         this.element.append(this.budgetForm);
+         this.container.append(this.element);
+         this.container.append(this.BudgetAndExpensesComponent);
+    }
+
+    createLablesAndInputs(item) {
+        const divFormBudgetInfo = createElement("div", {class: "div_form_budget_info" });
+        const lables = createElement("label", {class: "budget_form_label"});
+        lables.innerHTML = item.value;
+        const input = createElement("input", {type: item.type ,class: `${item.type}_${this.budgetForm.className}_input`, name: `${item.type}`});
```

Comment:
I do not think a input class on a input element should end with a 'input' string? please consider changing it.